### PR TITLE
[W5.11][W14-A1]Wong Peng Fai Shannon

### PR DESCRIPTION
**Pull request by shanwpf**
1st enhancement: Added `sort` command that sorts persons by alphabetical order
2nd enhancement: Display number of persons in address book after every command.

### CODE REVIEW

**Comment by muruges95** [comment]
Hi @shanwpf , nice use of Markdown in your comments! I also implemented a sort feature similar to yours so my one suggestion is possibly you could look to add in other functionality such as sort by email address or phone number or even tags! Other than that well done!

**Review by kexiaowen — COMMENTED** [review]
Good! Updated relevant parts in documentation (user guide) and added JUnit and I/O tests for your new features.

**Comment by shanmu9898** [comment]
Hey @shanwpf ! It's great that you took into account all the test cases including Junit testing and changed even I/O tests along with all the documentation. Your code is well abstracted looks like it is following the java coding standard nicely! Well done

File: src/seedu/addressbook/commands/SortCommand.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+public class SortCommand extends Command {
+
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Sorts the address book by alphabetical order.\n"
+            + "Example: " + COMMAND_WORD;
+
+    public static final String MESSAGE_SUCCESS = "Address book has been sorted!";
+
+    public SortCommand() {}
```
Comment (okkhoy) [inline src/seedu/addressbook/commands/SortCommand.java:15]: You don't need to explicitly provide an empty constructor; Java does it for you.

File: test/input.txt [hunk 2]
```diff
@@ -106,6 +106,14 @@
   # find multiple with some keywords
   find Charlie Betsy
 
+##########################################################
+# test sort command
+##########################################################
+
+  # sorts persons by alphabetical order
+  sort
```
Comment (okkhoy) [inline test/input.txt:114]: It would be nice to show a before and after effect of sort in your tests.

**Comment by okkhoy** [comment]
@shanwpf 
Some comments added. Please ack & close the PR after reading comments.

**Comment by shanmu9898** [comment]
yes sir! Noted! Thank you

**Pull request closed without merging**